### ISSUE TITLE
/api/clients does not return any name

### ISSUE THREAD

**Issue by yubiuser** [pi-hole/FTL#1768]
Reproduce: add any client with a `hostname` known to FTL: The tooltip should display the name as well

https://github.com/pi-hole/web/blob/52f198971ca0be75af05e83fe672f1ab1dbfd02e/scripts/pi-hole/js/groups-clients.js#L137-L161

`/api/clients`

```
{
    "clients": [
        {
            "client": "80:86:F2:9C:57:16",
            "name": null,
            "comment": null,
            "groups": [
                0
            ],
            "id": 6,
            "date_added": 1700565793,
            "date_modified": 1700565793
        }
    ],
    "took": 0.00057959556579589844
}
```

But `/api/clients/_suggestion does know the name

```
        {
            "hwaddr": "80:86:f2:9c:57:16",
            "macVendor": "Intel Corporate",
            "lastQuery": 1699914965,
            "addresses": "10.0.1.182",
            "names": "Thinkpad-Wifi.fritz.box"
        },
```

**Comment by DL6ER** [comment]
Trying to reproduce this locally isn't showing incorrect behavior right now for me...
![Screenshot from 2023-11-21 13-39-46](https://github.com/pi-hole/FTL/assets/16748619/3f87f703-d526-47e1-baa4-92350c4eca5c)

![Screenshot from 2023-11-21 13-39-07](https://github.com/pi-hole/FTL/assets/16748619/01c406a3-274d-48bf-9e82-c919a7a9f7b0)

but this is an `ip-...` pseudo-MAC, let me try again with a real hardware address.

**Comment by DL6ER** [comment]
I can confirm this behavior with clients specified by MAC address. The reason is that we don't store hostnames in the gravity database:
``` sql
CREATE TABLE client
(
        id INTEGER PRIMARY KEY AUTOINCREMENT,
        ip TEXT NOT NULL UNIQUE,
        date_added INTEGER NOT NULL DEFAULT (cast(strftime('%s', 'now') as int)),
        date_modified INTEGER NOT NULL DEFAULT (cast(strftime('%s', 'now') as int)),
        comment TEXT
);
```

The reason why `/api/clients/_suggestion` has is it is that it sources the entries from the `network` table (`pihole-FTL.db`) whereas `/api/clients/` is sourcing the clients from `client` in `gravity.db` where this is not available. It only works for clients specified by IP address because FTL internally resolves IP addresses to hostnames and shows this result.

**Comment by DL6ER** [comment]
See #1771
